Patches/SQLite3/CMakeLists.txt

```diff
@@ -0,0 +1,94 @@
+project ( SQLite3 )
+cmake_minimum_required(VERSION 2.8)
+
+# Build output paths
+if( NOT CMAKE_ARCHIVE_OUTPUT_DIRECTORY )
+  set( CMAKE_ARCHIVE_OUTPUT_DIRECTORY ${SQLite3_BINARY_DIR}/lib )
```

Comment:
It might be nice to use `GNUInstallDirs` in order to install to the correct platform-specific location